zdm/beams.py

```diff
@@ -27,14 +27,21 @@ def gauss_beam(thresh=1e-3,nbins=10,freq=1.4e9,D=64,sigma=None):
 		# uses 1.2 lambda on D
 		# check with Parkes: 1.38 GHz at 64m is 14 arcmin
 		HPBW=1.22*(constants.c/(freq*1e6))/D
-		sigma=(HPBW/2.)*(2*np.log(2))**0.5
+		sigma=(HPBW/2.)*(2*np.log(2))**-0.5
```

Comment:
this was a bug! This part of the code never got used, but still, best to change it...